src/index.js

```diff
@@ -49,44 +63,47 @@ const printList = () => {
   title.appendChild(titleText);
   title.appendChild(refresh);
   titleText.setAttribute('id', 'title');
-  const addTask = document.createElement('li');
+  const newTask = document.createElement('li');
   const addButton = document.createElement('button');
   addButton.setAttribute('id', 'add-button');
+  addButton.addEventListener('click', () => {
+    const task = document.querySelector('#add-task');
+    count += 1;
+    addTask(task.value, count);
+    printTask(task.value, count);
+    task.value = '';
+  });
   const arrow = new Image();
   arrow.src = Arrow;
   addButton.appendChild(arrow);
   const input = document.createElement('input');
   input.setAttribute('id', 'add-task');
   input.setAttribute('placeholder', 'Add to your list...');
-  addTask.appendChild(input);
-  addTask.appendChild(addButton);
+  newTask.appendChild(input);
+  newTask.appendChild(addButton);
   container.appendChild(title);
-  container.appendChild(addTask);
+  container.appendChild(newTask);
   tasksList.forEach((element) => {
-    printTask(element.description);
+    printTask(element.description, element.index);
   });
   const clearComplete = document.createElement('li');
   clearComplete.setAttribute('id', 'clear-complete');
   const clearButton = document.createElement('button');
   clearButton.textContent = 'Clear all completed';
+  clearButton.addEventListener('click', () => {
+    clearCompleted();
+  });
   clearComplete.appendChild(clearButton);
   container.appendChild(clearComplete);
 };
 
-const addTaskToLocalStorage = (description, index) => {
-  localStorage.setItem(index, JSON.stringify(new Task(description, index)));
-};
-
-const addTask = (description, index) => {
-  tasksList.push(new Task(description, index));
-  printTask(description, index);
-  addTaskToLocalStorage(description, index);
-};
-
 const addExamples = () => {
   addTask('Buy eggs', 1);
+  printTask('Buy eggs', 1);
   addTask('Wash the dishes', 2);
+  printTask('Wash the dishes', 2);
   addTask('Feed cats', 3);
+  printTask('Feed cats', 3);
 };
 
 const getLocalStorage = () => {
```

Comment:
**Kindly solve this issues: **
- I tried to refresh the page and i loose all my tasks in the list. If you check the console in your browser you will find an error related into your line 112.